### PR TITLE
NFTKey Marketplace (BNB) Integration with DappRadar

### PR DESCRIPTION
**Pull request by westonnelson**
NFTKey Marketplace (BNB) Integration with DappRadar - in light of the recent announcement of $RADAR incentives on ApeSwap, I noticed that NFTKey had not yet been integrated to DappRadar and have completed the necessary files for the BNB contract of the NFTKey Exchange. 

*Note - added both files as individual files in the ```src``` folder for your review.

Author: Weston Nelson
ENS:[ westonnelson.eth](https://etherscan.io/address/0xd131f1bcdd547e067af447dd3c36c99d6be9fdeb)
Twitter: https://twitter.com/westonnelson

I look forward to your feedback. Thank you!

### CODE REVIEW

**Comment by NamakeaAko** [comment]
BNB

**Comment by westonnelson** [comment]
Hello - I see the prior PR that I was working on from before is still included in this one.

My apologies if this caused any confusion - this is not relevant to this PR.

This is the committ hash I believe is needed to isolate what is relevant to the specific instance here: I hope this is useful for you @liutaurasjanuskas 

Hash: a5bbad360287b11923c1c757b73b31a1cf97e528

2 files: abi.json and index.ts for the BNB Chain Marketplace smart contract.

Please let me know if this provides any needed clarity and if you have any feedback!

Thanks again,

Weston

File: adapters/nft-key/index.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+class NFT_K {
+    // stands for NFT Key
+    constructor() {
+        this.name = "NFTK";
+        this.symbol = null;
+        this.token = "0x0000000000000000000000000000000000000000";
+        this.protocol = "AVAILABLE_PROTOCOLS.AVALANCHE";
+        this.block = 12855192;
+        this.contract = "0x1a7d6ed890b6c284271ad27e7abe8fb5211d0739";
+        this.events = struct Listing {
```
Comment (Mintiss) [inline adapters/nft-key/index.js:20]: The events need to be just event names, not structs from the contract.
Please make sure that the provider passes the tests

**Review by Mintiss — CHANGES_REQUESTED** [review]

File: adapters/nft-key/index.js (hunk 1 above)
Comment (Mintiss) [reply adapters/nft-key/index.js:20]: Feel free to look at our already integrated providers from the main branch as an example

Comment (earnadvise) [reply adapters/nft-key/index.js:20]: https://github.com/dappradar/nft-sales-adapters/issues/89#issue-1354518424

**Comment by earnadvise** [comment]
sudoswap

**Comment by liutaurasjanuskas** [comment]
Provider is not working and contributor stopped responding.

**Pull request closed without merging**